src/codegen/mod.rs

```diff
@@ -3434,6 +3434,14 @@ impl TryToRustTy for Type {
 
                 let inner =
                     inner.into_resolver().through_type_refs().resolve(ctx);
+                let is_objc_pointer = if let Some(ty) = inner.kind().as_type() {
```

Comment:
nit: Use `map_or(false`? But actually this should be moved down where we have `inner_ty`, which is basically `inner` unwrapped to a type.